src/main/java/ulcrs/controllers/CourseController.java

```diff
@@ -35,7 +29,7 @@ public RouteGroup routes() {
 
     private Course getCourse(Request request, Response response) {
         response.type(CONTENT_TYPE_JSON);
-        int id = Integer.valueOf(request.params("id"));
+        int id = Integer.valueOf(request.params(":id"));

```

Comment:
Does this still work with the colon?

Reply:
Yes, and that's on official documentation

Reply:
Sounds good to me